src/ipts/protocol/dft.hpp

```diff
@@ -15,8 +15,10 @@ constexpr u8 PRESSURE_ROWS = 6;
 
 enum class Type : u8 {
 	Position = 6,
+	Position2 = 7,
 	Button = 9,
 	Pressure = 11,
+	Dft0x0a = 0x0a,
```

Comment:
```suggestion
	Position = 0x6,
	Position2 = 0x7,
	Button = 0x9,
	Button2 = 0xA,
	Pressure = 0xB,
```

You could also consider putting the relation to MPP into the name, e.g. `PositionMPP_1` and `PositionMPP_2`.